server/sendemail.ts

```diff
@@ -0,0 +1,51 @@
+import { SES } from "aws-sdk";
+import { SendEmailRequest } from "aws-sdk/clients/ses";
+
+export const sendEmail = (email: string) => {
+  try {
+    const REGION = "us-east-1";
+
+    const charset = "UTF-8";
+
+    const html = `<html>
+      <head>
+      <title>Page Title</title>
+      <style>
+      </style>
+      </head>
+      <body>
+      <div style=font-family: Arial, Helvetica, sans-serif;">
+      <img src="kcsa_logo.png" alt="KCSA Logo" style="width:200px">
+      <h1>Welcome to KCSA!</h1>
+      <p>You have created an account with us at KCSA. To verify your email, please click the link below.</p>
+      <a href="http://kcsacanada.ca/">http://kcsacanada.ca/</a>
+      
+      </div>
+      </body>
+      </html>`;
+
+    const ses = new SES({ region: REGION });
+
+    const params: SendEmailRequest = {
+      Source: "koreancanadiansoccer@gmail.com",
+      Destination: {
+        ToAddresses: [email],
+      },
+      Message: {
+        Subject: {
+          Data: "Welcome to KCSA",
+          Charset: charset,
+        },
+        Body: {
+          Html: {
+            Data: html,
+            Charset: charset,
+          },
+        },
+      },
+    };
+    ses.sendEmail(params).promise();
```

Comment:
why .promise? 
Make this function `async` and just put `await` here?

Reply:
ok, i'll try that

Reply:
Also this email should be more dynamic.
We will have few different emails being sent out - but you can get that done later

Reply:
okok

Reply:
I tried async with await but it still can't send email for some reason.
I don't know why it only sends when I put promise() at the end.

Even tried putting the credentials from .env as parameters and still didn't work without promise.
```
const ses = new SES({
      accessKeyId: process.env.AWS_ACCESS_KEY_ID,
      secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
      region: REGION,
    });
```

Reply:
oh wait looks like `promise()` it self is callback function. 

Just leave it like this then.